pyV2DL3/eventdisplay/util.py

```diff
@@ -96,19 +96,19 @@ def getGTI(BitArray, run_start_from_reference):
 
     if TimeArray_b[0] != "0":
         gti_start.append(0)
-        
-    for i in range(1, duration_s-1):
-        
+
+    for i in range(1, duration_s - 1):
+
         if (TimeArray_b[i] == "0") & (TimeArray_b[i - 1] == "1"):
```

Comment:
Here and in the following: I think python wants to see a 'and' and not a '&' for boolean operators.